std_msgs/UInt16 pallet
std_msgs/UInt16 no
---
bool success

